samples/net/sockets/echo_server/overlay-nsos.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_ETH_NATIVE_POSIX=n
```

Comment:
(sidetrack) (@jukkar ) I wonder if after we have this new driver, if it we should change defaults. What do you think?

Reply:
> I wonder if after we have this new driver, if it we should change defaults. What do you think?

@aescolar What defaults do you mean, could you elaborate a bit more?

Reply:
I guess that I was thinking that instead of this default https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/posix/native_sim/Kconfig.defconfig#L22
we'd default to this driver `if NETWORKING`. (But you know much more about this kconfigs and what is better to test more or offer by default, so it was just an open question)

Reply:
> I guess that I was thinking that instead of this default https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/posix/native_sim/Kconfig.defconfig#L22

That is certainly possible, but that needs to be discussed separately after we have merged this and seen how to use it. After all, this change is somewhat specialized as it bypasses L2 so it depends on use cases when to use it or not.